xtask/src/main.rs

```diff
@@ -33,7 +35,11 @@ enum TestCommand {
     TestCross,
     TestHost,
     TestLint,
-    TestSnapshot,
+    TestSnapshot {
```

Comment:
Let's add a small text, so people know about this option, when running `cargo xtask --help`. I don't have strong feelings about the wording, but this is my suggestion:
```suggestion
    /// Run snapshot tests or optionally update the expected output
    TestSnapshot {
```